### PR TITLE
[Bug Fix] User reactions

### PR DESCRIPTION
**Pull request by Simpcyclassy**

### CODE REVIEW

File: components/footer/AppFooter.jsx [hunk 1]
```diff
@@ -14,21 +14,21 @@ class AppFooter extends React.Component {
         return (
             <div style={CONTAINER}>
                 <Menu
-                    mode={HORIZONTAL}
-                    theme={DARK}
-                    style={MAIN_NAV}
+                  mode={HORIZONTAL}
+                  theme={DARK}
+                  style={MAIN_NAV}
```
Comment (enyeInc) [inline components/footer/AppFooter.jsx:19]: this spacing change is wrong

File: components/home/reducers.js [hunk 2]
```diff
@@ -23,11 +24,13 @@ function generateUploadsMap() {
...
     const updatedUploads = new Map([...uploads.entries()]);
     const userUpload = updatedUploads.get(id);
-    if (!userUpload.reactions[type]) {
+
+    if (USER && !userUpload.reactions[type]) {
         userUpload.reactions[type] += 1;
     } else {
         userUpload.reactions[type] -= 1;
     }
+
```
Comment (enyeInc) [inline components/home/reducers.js:53]: This doesn't fix your bug chioma

**Comment by enyeInc** [comment]
@Simpcyclassy this PR is confusing. Why are you adding `AppFooter/Header` to a PR called `bug/Chioma/user_uploads`. Can you create an PR with just the bug fix and nothing else? For the rest of the changes just create another PR, get the context and code around a PR the same

File: components/footer/AppFooter.jsx (hunk 1 above)
Comment (Simpcyclassy) [reply components/footer/AppFooter.jsx:19]: Alright. Will fix it

**Comment by enyeInc** [comment]
@Simpcyclassy is this ready to be reviewed?

**Pull request merged**